specification/sections/05-events.md

```diff
@@ -28,9 +28,16 @@ graph
 
 > The `provider` **MAY** define a mechanism for signaling the occurrence of one of a set of events, including `PROVIDER_READY`, `PROVIDER_ERROR`, `PROVIDER_CONFIGURATION_CHANGED` and `PROVIDER_STALE`, with a `provider event details` payload. 
 
+Providers cannot emit `PROVIDER_CONTEXT_CHANGED` or `PROVIDER_CONTEXT_PENDING` event. These are emitted only by the SDK during context reconciliation.
+
 If available, native event-emitter or observable/observer language constructs can be used.
 
-see: [provider event types](../types.md#provider-events), [`event details`](../types.md#provider-event-details).
+When a provider is unable to evaluate flags (perhaps due to loss of connection with a remote service) the provider can signal this by emitting a `PROVIDER_ERROR` event.
+If the error state is irrecoverable, the `PROVIDER_STATE_FATAL` error code can be used.
```

Comment:
Is it possible to set `PROVIDER_STATE_FATAL` during initialization? This may happen if the access key is invalid.

Reply:
Yes, that's one of the main purposes. I can mention it in the init section as well.

Reply:
I've done this.